app/src/main/AndroidManifest.xml

```diff
@@ -96,7 +96,7 @@
 
         <activity
             android:name=".activity.MainActivity"
-            android:configChanges="keyboardHidden|orientation|screenSize|smallestScreenSize|screenLayout|density|uiMode|keyboard|navigation"
+            android:configChanges="keyboardHidden|orientation|smallestScreenSize|screenLayout|density|uiMode|keyboard|navigation"
```

Comment:
Hmm, the thing is, this line is there for a reason. Removing it makes rotating the screen significantly less pleasant because it takes such a long time. On my (rather large) test database, rotating the app on the `develop` branch takes 2 seconds. On your branch, it takes 4 seconds and the screen flickers. There must be a way to avoid the long loading times and still fix the problems I mentioned in #5920.